specification/network/resource-manager/Microsoft.Network/stable/2018-05-01/ExpressRouteCircuitARPTableList.json

```diff
@@ -0,0 +1 @@
+{
  "parameters": {
    "expressRouteCircuitName": "circuitName",
    "resourceGroupName": "rg1",
    "api-version": "2017-11-01",
    "subscriptionId": "subid",
    "peeringName": "peeringName",
    "devicePath": "devicePath"
  },
  "responses": {
    "200": {
        "value": [
         {
            "age": 0,
            "interface": "Microsoft",
            "ipAddress": "IPAddress",
            "macAddress": "macAddress"
         }
        ]
    },
    "202": {
        "value": [
         {
            "age": 0,
            "interface": "Microsoft",
            "ipAddress": "IPAddress",
            "macAddress": "macAddress"
         }
        ]
    }
  }       
}

```

Comment:
Please configure your git to check in Unix style line endings.

Reply:
@gimotwanMSFT  was it done?